### PR TITLE
Feat: emit signals on profile showcase preferences and save actions

### LINKED ISSUE #13968: [Profile Showcase] Emit signal when preferences have been correctly saved AND when there's a failure
### Description

According to the flow described in the partner UI task [here](https://github.com/status-im/status-desktop/issues/13950), the backend should emit 2 different signals in response of the preferences save action:
1. When save actions has been succeded. 
2. When save action has failed

UI will expect these 2 signals from `profileSectionModule` and will listen to them from `ProfileSectionStore`.

### PR DESCRIPTION
**Pull request by MishkaRogachev**
Close #13968

### What does the PR do

- [x] Emit succeeded and failed signals while saving profile identity
- [x] Emit succeeded and failed signals while saving profile showcase preferences

### Affected areas

Profile

### CODE REVIEW

**Review by caybro — APPROVED** [review]
LGTM :+1:

**Review by noeliaSD — APPROVED** [review]
Perfect! Thanks!!!

**Comment by noeliaSD** [comment]
**Summary of my test:**
- Tested both signals and they are correctly emited (failure was forced by sending repeated accounts).
- We need 2 additional signals to cover the identity tab changes (as discussed offline, we expect @MishkaRogachev  adds them in this pr).
- It seems `ProfileStore` is instanciated multiple times so we will need to resolve this (I'll take a look at it in [here](https://github.com/status-im/status-desktop/issues/13950)).

File: src/app_service/service/profile/service.nim [hunk 1]
```diff
@@ -86,27 +86,29 @@ QtObject:
...
         error "could not set display name"
-        return
+        return false
       if(not self.settingsService.saveDisplayName(displayName)):
         error "could save display name to the settings"
-        return
+        return false
+      return true
     except Exception as e:
       error "error: ", procName="setDisplayName", errName = e.name, errDesription = e.msg
```
Comment (caybro) [inline src/app_service/service/profile/service.nim:113]: What is returned here in case of an exception?

Comment (MishkaRogachev) [reply src/app_service/service/profile/service.nim:114]: fixed

**Pull request merged**